README.rst

```diff
@@ -14,18 +14,78 @@ For full details see http://figure.openmicroscopy.org
 
 
 Requirements
-------------
+============
 
-* OMERO 4.4.x or OMERO 5.x
+* OMERO 5.2.6 or newer.
 
 Installation
-------------
+============
+
+Install OMERO.web.
+
+This app installs into the OMERO.web framework.
+
+::
+
+    $ pip install omero-figure
+
+Add figure custom app to your installed web apps:
+
+::
+
+    $ bin/omero config append omero.web.apps '"omero_figure"'
+
+Display a link to 'Figure' at the top of the webclient:
+
+::
+
+    $ bin/omero config append omero.web.ui.top_links '["Figure", "figure_index", {"title": "Open Figure in new tab", "target": "figure"}]' 
+
+Now restart OMERO.web as normal.
+
+**Warning**:
+
+OMERO.figure version 2.x is meant to be installed into OMERO.web **5.2.6 or newer**.
+This is due to Django Framework compatibility and a package re-organization required to distribute the application using Python Package Manager PyPi.
```

Comment:
I'm not sure you need to mention Django or Pypi here?
I would say: "OMERO.figure version 2.x requires OMERO.web **5.2.6 or newer**. OMERO.figure was reorganized in version 2.0 to make it ```pip``` installable and the new package structure is not compatible with older OMERO.web versions.